hardware/computemodule/cmio-display.md

```diff
@@ -47,26 +63,25 @@ Please note that the wiring is slightly different from that on the Camera page,
 	GPIO31 - CAM0_IO0
 	```
 
-	![GPIO connection for a single display and two Camera Modules](images/CMIO-Cam-Disp-GPIO.jpg)
-	(Please note this image needs to be updated to show two Camera Modules, or have the extra jumper leads removed)
-
-1. On the Compute Module, for the display and one Camera Module, run:
+1. (CM4 only) Add jumpers to J6.
 
-	```sudo wget https://goo.gl/gaqNrO -O /boot/dt-blob.bin```  
-
-  For the display and two Camera Modules, run:
+1. Power up the Compute Module and run:
 
 	```sudo wget https://goo.gl/htHv7m -O /boot/dt-blob.bin```
 
 1. Reboot for the `dt-blob.bin` file to be read.
 
 	![Camera Preview on the 7 inch display](images/CMIO-Cam-Disp-Example.jpg)
-	(Please note this image needs to be updated to show two Camera Modules, or have the extra jumper leads removed)
+	(Please note this image needs to be updated to show two Camera Modules and the standard wiring)
 
 ### Software support
 
 There is no additional configuration required to enable the touchscreen. The touch interface should out work of the box once the screen is successfully detected.
```

Comment:
I know its not in the change, but there is an extra 'out' in here

Reply:
Fixed. The "out" is needed, but is misplaced by a word (off-by-one error!)